app/react/src/server/config/webpack.config.prod.js

```diff
@@ -29,7 +29,7 @@ export default function() {
           screw_ie8: true,
           warnings: false,
         },
-        mangle: false,
+        mangle: true,
```

Comment:
I think this is okay because its the prod config that compiles a static bundle. Presumably, users wouldn't be debugging from a static file served on an http host somewhere.